conanfile.py

```diff
@@ -39,19 +39,20 @@ def set_version(self):
     def requirements(self):
         self.tool_requires("cmake/[>=3.19]")
         self.requires("fmilibrary/[~2.3]")
-        self.requires("libzip/[>=1.7 <1.10]") # 1.10 deprecates some functions we use
+        self.requires("libcbor/0.11.0")
+        self.requires("libzip/[~1.11]")
         self.requires("ms-gsl/[>=3 <5]", transitive_headers=True)
+        self.requires("boost/[~1.85]", transitive_headers=True, transitive_libs=True)  # Required by Thrift
         if self.options.proxyfmu:
-            self.requires("proxyfmu/0.3.2@osp/stable")
-            self.requires("boost/[~1.81]", transitive_headers=True, transitive_libs=True) # Required by Thrift
-        else:
-            self.requires("boost/[>=1.71]", transitive_headers=True, transitive_libs=True)
+            self.requires("proxyfmu/0.3.3@osp/testing-feature_779-state-serialization",
+                          transitive_headers=True,
+                          transitive_libs=True)
         self.requires("yaml-cpp/[~0.8]")
         self.requires("xerces-c/[~3.2]")
 
     # Exports
     exports = "version.txt"
-    exports_sources = "*"
+    exports_sources = "*", "!cmake-build*", "!.venv", "!.git"
```

Comment:
Why is this change necessary? Do these files/dirs get created during the CI build process and end up in the packaged artifacts?